src/share/timing/GPTLget_memusage.c

```diff
@@ -118,7 +118,7 @@ int GPTLget_memusage (int *size, int *rss, int *share, int *text, int *datastack
   */
 
   pid = (int) getpid ();
-  if (pid > 999999) {
+  if (pid > 9999999) {
```

Comment:
Curious why this check is needed?

Looking at the man page (https://man7.org/linux/man-pages/man2/getpid.2.html), this call should always be successful at least on Linux. Perhaps check for a positive value or what's the corner case we are trying to handle?

Reply:
Updated